tokens/sass-helpers.js

```diff
@@ -14,7 +14,7 @@ const chroma = require('chroma-js');
  * @return chroma-js color instance (one of dark or light variants)
  */
 function colorYiq(color, light, dark, threshold) {
-  const defaultsFile = fs.readFileSync(path.resolve(__dirname, 'src', 'global', 'other.json'), 'utf8');
+  const defaultsFile = fs.readFileSync(path.resolve(__dirname, 'src', 'themes/light/global', 'other.json'), 'utf8');
```

Comment:
[curious] Is everything in the `other.json` file specific to the light theme variant? Or are some of the properties, possibly like `theme-interval`, and `yiq-contrasted-threshold`.

Reply:
yes, `theme-interval` and `yiq-contrasted-threshold` are not theme-specific, moved the to the `core` tokens